# 再生停止リクエスト
string playback_id           # 再生ID

---
# 再生停止レスポンス
bool success                 # 成功フラグ
string message              # メッセージ 